python/graphstorm/model/loss_func.py

```diff
@@ -18,6 +18,7 @@
 import torch as th
 from torch import nn
 import torch.nn.functional as F
+import torchvision
```

Comment:
Is torchvision part of our dependency list?

Reply:
I need to check.

Reply:
torchvision is not in our dependency list.
I have updated the implementation.

Reply:
OK can we remove this line then?